src/index/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '../core'
```

Comment:
Is the `/index/` directory necessary? This is not pointed at anywhere.

Reply:
Thank you!
I don't fully understand how `bunchee` works, so I thought this was unnecessary because `package.json` points `core/*` files in the `exports` field. But this is necessary to generate `dist/core/*` files.
When  I removed `/index/`, the build files weren't created.